purchase_requisition_auto_rfq/models/purchase_requisition.py

```diff
@@ -61,3 +72,37 @@ def _create_purchase_order(self, seller_id):
         po = self.env["purchase.order"].with_context(**ctx).create(vals)
         po._onchange_requisition_id()
         return po
+
+    def action_all_rfq_related_send(self):
+        self.ensure_one()
+        rfqs = self.purchase_ids.filtered(lambda x: x.state != "cancel")
+        template_id = self.env.ref("purchase.email_template_edi_purchase", False).id
+        ctx = dict(
+            self.env.context,
+            default_model="purchase.order",
+            active_model="purchase.order",
+            active_id=rfqs.ids[0],
+            active_ids=rfqs.ids,
+            default_template_id=template_id,
```

Comment:
... and also change this:

```suggestion
            default_template_id=template and template.id,
```